commit 35c41f35b08e981caa062f963ad2611ee6ef35a9
Author: Timothy Pearson <kb9vqf@pearsoncomputing.net>
Date:   1343320733 -0500

    Fix a parallel build bug
    Thanks to 'Nix' for the patch!

diff --git a/filesharing/advanced/propsdlgplugin/CMakeLists.txt b/filesharing/advanced/propsdlgplugin/CMakeLists.txt
index f6477d1..26d94f5 100644
--- a/filesharing/advanced/propsdlgplugin/CMakeLists.txt
+++ b/filesharing/advanced/propsdlgplugin/CMakeLists.txt
@@ -32,6 +32,7 @@ install( FILES
 
 tde_add_library( propsdlgplugin_common STATIC_PIC AUTOMOC
   SOURCES propertiespage.cpp propertiespagegui.ui
+  DEPENDENCIES filesharesamba
 )
 
 
